src/solvers/dgsem_t8code/containers_2d.jl

```diff
@@ -0,0 +1,57 @@
+@muladd begin
+
```

Comment:
```suggestion
@muladd begin
#! format: noindent

```
Please run the formatter again after this and add the comment on `@muladd` that we use elsewhere